src/core/api/details/details.service.ts

```diff
@@ -18,7 +18,7 @@ export class DetailsService {
     private transformDataDetailsService: TransformDataDetailsService
   ) {}
 
-  getDataForWeatherTable(): Observable<DetailsWeather | any> {
+  getDataForWeatherTable(): Observable<DetailsInfo | any> {
```

Comment:
DetailsInfo | undefined i think

Reply:
Its ok if it will unknown? "Observable<DetailsInfo | unknown>"